routes/api/profile.js

```diff
@@ -4,6 +4,8 @@ const auth = require('../../middleware/auth');
 const { check, validationResult } = require('express-validator');
 
 const Profile = require('../../models/Profile');
+const User = require('../../models/User');
+const Post = require('../../models/Post');

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/spumony/xvent-project-backend/pullRequest?prid=7531905)
```suggestion
const Post = require("../../models/Post");
```